tests/unit/wait.tcl

```diff
@@ -175,10 +175,39 @@ tags {"wait aof network external:skip"} {
             $replica config set appendfsync everysec
 
             test {WAITAOF replica copy everysec} {
+                $replica config set appendfsync everysec
+                waitForBgrewriteaof $replica ;# Make sure there is no AOFRW
+
                 $master incr foo
                 assert_equal [$master waitaof 0 1 0] {1 1}
             }
 
+            test {WAITAOF replica copy everysec with AOFRW} {
+                $replica config set appendfsync everysec
+
+                # Need an AOFRW that can be done in a second.
+                $replica bgrewriteaof
```

Comment:
i understand this test has some (high) chance of hitting the race, but it's not guaranteed, right?
maybe we need to slow down the rewrite to make sure it'll not finish before the WAITAOF?

Reply:
i will re-think it later, and see how to improve the test and avoid some races. these tests were written in a hurry.